src/hb-ot-layout.cc

```diff
@@ -250,6 +306,22 @@ hb_ot_layout_get_glyphs_in_class (hb_face_t                  *face,
   return face->table.GDEF->table->get_glyphs_in_class (klass, glyphs);
 }
 
+
+/**
+ * hb_ot_layout_get_attach_points:
+ * @face: The #hb_face_t to work on
+ * @glyph: The #hb_codepoint_t code point to query
+ * @start_offset: The offset into the attachment-point array to start returning
+ * @point_count: (in) The number of attachment points to return
+ *               (out) The number of attachment points returned in the array
```

Comment:
The gobject-introspection syntax should be `@point_count: (inout):`, the text would be something like “size of point array as input, filled size as output”.

Reply:
OK; I had wondered about that. Will do.